ttnn/cpp/ttnn/operations/moreh/moreh_mean_backward/moreh_mean_backward_pybind.cpp

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_mean_backward_pybind.hpp"
+
+#include "pybind11/decorators.hpp"
+#include "ttnn/operations/moreh/moreh_mean_backward/moreh_mean_backward.hpp"
+
+namespace ttnn::operations::moreh::moreh_mean_backward {
+void bind_moreh_mean_backward_operation(py::module &module) {
+    bind_registered_operation(
+        module,
+        ttnn::moreh_mean_backward,
+        "Moreh moreh_mean_backward Operation",
+        ttnn::pybind_arguments_t{
+            py::arg("output_grad"),
+            py::kw_only(),
+            py::arg("dim"),
+            py::arg("keepdim"),
+            py::arg("input_grad_shape") = std::nullopt,
```

Comment:
can't one extract shape from a tensor below?

Reply:
Since the input_grad tensor is `optional<tensor>`, it's not always available.